crates/ursa-network/src/behaviour.rs

```diff
@@ -159,10 +164,40 @@ impl<P: StoreParams> Behaviour<P> {
                 if relay_client.is_none() {
                     panic!("relay client not instantiated");
                 }
-                dcutr::behaviour::Behaviour::new()
+                Dcutr::new()
             })
             .into();
 
+        let mdns = config
```

Comment:
Revert

Reply:
all the other variables in this function are using `then(||` syntax, what do?

Reply:
Close the pr